src/front/app/main.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { render } from 'react-dom';
 import { Provider } from 'react-redux';
 import { Router } from 'react-router';
-import appStore from './store';
+import appStore from './store/store';
```

Comment:
I think that it will be handy to rename *./store/store.js* into *./store/index.js*
this will allow to keep import statement the same as it was
to avoid such duplications in path